src/services/OfferService.js

```diff
@@ -42,14 +42,14 @@ export default class OfferService {
     return token
   }
 
-  getAllOffers (): Promise<void> {
+  getAllOffers (): Promise<Array<Offer>> {
     return Offer.find()
       .select('-_id -__v')
       .populate({path: 'formData', select: '-_id -__v'})
```

Comment:
Are you missing a `.lean()` here for consistency?

Reply:
Added it